src/nvim/ex_docmd.c

```diff
@@ -6806,36 +6809,57 @@ void free_cd_dir(void)
 
 #endif
 
-/*
- * Deal with the side effects of changing the current directory.
- * When "local" is TRUE then this was after an ":lcd" command.
- */
-void post_chdir(int local)
+/// Deal with the side effects of changing the current directory.
+///
+/// @param scope  Scope of the function call (editor, tab or window).
+void post_chdir(CdScope scope)
 {
+  assert(scope <= MAX_CD_SCOPE);
```

Comment:
compiler will warn about this implicitly because the switch statements don't have a `default` case.